### PR TITLE
Replace 'd-installer' with 'installer'

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: tasks/run-yast-with-podman.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <title>Introduction</title>
     <para>
       This article describes how to start the &yast; workload on the
```
Comment (dariavladykina) [inline tasks/run-yast-with-podman.xml:25]:
```suggestion
      This article describes how to start the &yast; workload on
```

File: concepts/alp-installer-script-based-automated-deployment.xml [hunk 2]
```diff
@@ -13,10 +13,10 @@
  xmlns:trans="http://docbook.org/ns/transclusion">
   <info>
     <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
-    <title>Script-based automated deployment with &dinstaller;</title>
+    <title>Script-based automated deployment with the &alp-installer; installer</title>
     <abstract>
       <para>
-        The default &productname; (&productnameshort;) installer &dinstaller;
+        The default &productname; installer &alp-installer;
```
Comment (dariavladykina) [inline concepts/alp-installer-script-based-automated-deployment.xml:19]:
```suggestion
        The default &productname; installer, &alp-installer;,
```

File: tasks/deploy-alp-installer.xml [hunk 3]
```diff
@@ -9,27 +9,27 @@
...
   <info>
     <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
-    <title>Deploying &productnameshort; interactively using &dinstaller;</title>
+    <title>Deploying &productnameshort; interactively using the &alp-installer; installer</title>
     <abstract>
       <para>
-        This article describes how to deploy the &productname;
-        (&productnameshort;) interactively using &dinstaller;. For an automated
-        deployment, refer to <xref linkend="agama-automated-deployment"/>.
+        This article describes how to deploy the &productnameshort;
```
Comment (dariavladykina) [inline tasks/deploy-alp-installer.xml:23]:
```suggestion
        This article describes how to deploy &productnameshort;
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions. Thanks!

**Comment by tbazant** [comment]
merging manually

**Pull request closed without merging**